apps/melding-form/src/app/(map)/locatie/kies/_components/SideBarTop/SideBarTop.module.css

```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-direction: column;
   gap: var(--ams-space-m);
-  padding-block: var(--ams-space-m);
+  padding-block-start: var(--ams-space-m);
```

Comment:
This also remove padding bottom from Sidebar Top on narrow screens, which we don't want:

<img width="704" height="182" alt="sidebar no padding" src="https://github.com/user-attachments/assets/85cd0765-0d4e-4507-a6a3-1cf12e517398" />